### ISSUE TITLE
infrequent blank screen upon Equinum use - 0.9.2

### ISSUE THREAD

**Issue by Zeikfried** [herp-a-derp/Corruption-of-Champions#706]
On one of my save files I noticed that use of Equinum occasionally results in a blank screen, perhaps one in five or one in ten uses. Any stat changes are still represented faithfully on the stat sidebar. The character in question has the Alchemist history and was played by using every usable item found, either of which may be causing the issue. I have retained a copy of the save at the point I first noticed the bug, if needed.

**Comment by IndirectOne** [comment]
Could you narrow it down?
Is this happening when you're in camp, in a dungeon or after combat?
There are two ways into the inventory system - inventoryMenu, which shows when you're in camp and takeItem, which is called when you defeat something in combat and get an item.

**Comment by Zeikfried** [comment]
I've observed it in camp and post-battle.

**Comment by IndirectOne** [comment]
Yeah, if you could send me the save that would help. I've guzzled equinum several times with a couple of characters and had nothing but the bad end trigger. I can't see where it would miss popping up a menu, it must be breaking or looping infinitely somewhere inside the equinum function itself.

**Comment by Zeikfried** [comment]
The menu itself works. Only the text display is absent. And I don't have contact info for you.

**Comment by IndirectOne** [comment]
Hmmm. That's a different problem, though after checking through equinum again I still can't see what might cause it. My email is kquesom@gmail.com. Sending me that save is probably the only way to find this bug.

**Comment by Fenoxo** [comment]
The forum should allow attachments of save files nowadays. You could dump
it in a thread and link it.

**Comment by IndirectOne** [comment]
Well I'm afraid I can't reproduce that bug. Loaded that save in 0.9.2, I could chug equinum constantly and each time I got at least a few lines of text.

I doubt your version of firefox is at fault, but it probably wouldn't hurt to update it and also upgrade adobe AIR and flash to the latest versions. Also, maybe re-download the 0.9.2 file, maybe it's corrupt. Another thing to try - how about loading it in CoC-Alpha? Maybe whatever was causing it got fixed by some partially unrelated code change.
